src/state_machine.zig

```diff
@@ -484,6 +485,10 @@ pub fn StateMachineType(
             for (accounts) |*a| {
                 self.forest.grooves.accounts_immutable.prefetch_enqueue(a.id);
             }
+
+            // Set the union tag so access via &self.prefetch_context.* doesn't trap.
+            self.prefetch_context = .{ .accounts_immutable = undefined }; 
```

Comment:
Maybe use "Activate" word instead of "Set", as "active union variant" seems to be more specific bit of terminology.